auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.java

```diff
@@ -1283,4 +1339,9 @@ public void shouldParseUnauthorizedPKCEError() throws Exception {
         }.getType();
         return gson.fromJson(request.getBody().readUtf8(), mapType);
     }
+
+    private String getDefaultLocale() {
```

Comment:
wouldn't it be better if we add a test for the actual function  https://github.com/auth0/Auth0.Android/pull/15/files#diff-f8d8baa1565ca2a3ec49d45e48f3febdR144
and then use that same one?